packages/@aws-cdk/aws-location-alpha/test/map.test.ts

```diff
@@ -0,0 +1,136 @@
+import { Match, Template } from 'aws-cdk-lib/assertions';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import { Stack } from 'aws-cdk-lib';
+import { CustomLayer, Map, PoliticalView, Style } from '../lib';
+
+let stack: Stack;
+beforeEach(() => {
+  stack = new Stack();
+});
+
+test('create a map', () => {
+  new Map(stack, 'Map', {
+    mapName: 'my-map',
+    description: 'my map for test',
+    style: Style.VECTOR_ESRI_NAVIGATION,
+    customLayers: [CustomLayer.POI],
+    politicalView: PoliticalView.INDIA,
+  });
+
+  Template.fromStack(stack).hasResourceProperties('AWS::Location::Map', {
+    MapName: 'my-map',
+    Description: 'my map for test',
+    Configuration: {
+      CustomLayers: ['POI'],
+      PoliticalView: 'IND',
+      Style: 'VectorEsriNavigation',
+    },
+  });
+});
+
+test('creates a map with empty description', () => {
+  new Map(stack, 'Map', {
+    description: '',
+    style: Style.VECTOR_ESRI_NAVIGATION,
+  });
+
+  Template.fromStack(stack).hasResourceProperties('AWS::Location::Map', {
+    Description: '',
+  });
+});
+
+test('throws with invalid description', () => {
+  expect(() => new Map(stack, 'Map', {
+    description: 'a'.repeat(1001),
+    style: Style.VECTOR_ESRI_NAVIGATION,
+  })).toThrow('`description` must be between 0 and 1000 characters, got: 1001 characters.');
+});
+
+test('throws with invalid name', () => {
+  expect(() => new Map(stack, 'Map', {
+    mapName: 'inv@lid',
+    style: Style.VECTOR_ESRI_NAVIGATION,
+  })).toThrow('`mapName` must contain only alphanumeric characters, hyphens, periods and underscores, got: inv@lid.');
+});
+
+test('throws with invalid name length', () => {
```

Comment:
It seems that the following test may overlap in content!

Reply:
Thanks. updated.